app/jobs/submit_symphony_request_job.rb

```diff
@@ -85,6 +90,15 @@ def barcodes
       items
     end
 
+    def copy_notes
+      return if request.public_notes.blank?
+      result = []
+      request.public_notes.each_pair do |name, val|
```

Comment:
This kind of confused me.  Few questions.  Why use `each_pair` here?  Would `each` be more idiomatic?

Also, the block variables may be more clear as `barcode` and `note`?